### DISCUSSION TITLE
Load Map Center

### DISCUSSION THREAD

**Discussion by jwuerth** [vradarserver/vrs#120] [Q&A]
Hello,
I am running VRS on a Linux machine. I am using a public URL to link into my instance externally. the URL is [https://adsb.wuerth.contact](https://adsb.wuerth.contact). Upon first loading it is showing the location to be somewhere in Antarctica. I have created a custom content file and am pointing to that file correctly from the VRS Web Admin page. The code is set to be at the END of the HEAD per the instructions from the documentation. The code I am inserting is as follows;

#### 
    if(VRS && VRS.globalDispatch && VRS.serverConfig) {
        VRS.globalDispatch.hook(VRS.globalEvent.bootstrapCreated, function(bootStrap) {
          VRS.globalDispatch.currentLocationFixed = { lat: 41.888, lng: -87.794 };
        });
    }
### 

Even with this script inserted, the map is not loading at the correct location. Any suggestions would be welcomed.

**Comment by vradarserver** [comment]
The initial location, including the setting you're assigning there, is only used when a *new* browser visits the site. After the first visit the position is stored in the browser's local storage and reused on subsequent visits.

If you had visited the site with the browser you're testing with before you set up your custom script then the script will have no effect, the browser will reuse whatever position you left it at on your previous visit.

If you want to test initial settings then you need to go to the settings page (settings.html) and click REMOVE ALL to clear all of VRS's settings from the browser's local storage.

However, you don't need a custom script to set the browser's initial location. It can be configured - along with a bunch of other settings - by following the instructions here:

https://virtualradarserver.co.uk/Documentation/WebServer/WebSiteOptions.aspx#initial-settings

It's a bit clunky but it will work.
